tests/ui-toml/undocumented_unsafe_blocks/undocumented_unsafe_blocks.rs

```diff
@@ -0,0 +1,534 @@
+//@aux-build:proc_macro_unsafe.rs
```

Comment:
A small site note @Centri3, AFAIK you requested that this file is the same as the normal test file. Generally speaking, I think it's fine to have some basic tests for the lint, some *don't lint* examples and then the specific case, which is influenced by the config value. Having a lot of output in the `.stderr` file can make it hard to distinguish the actual changes.

This is not to imply that either variant is better. This is up to the reviewer to decide and since you mainly reviewed this one, I'll follow your lead. Just so you know, it's not mandatory :)